stubs/tensorflow/tensorflow/_aliases.pyi

```diff
@@ -29,6 +29,7 @@ KerasSerializable: TypeAlias = KerasSerializable1 | KerasSerializable2
 
 Slice: TypeAlias = int | slice | None
 FloatDataSequence: TypeAlias = Sequence[float] | Sequence[FloatDataSequence]
+IntDataSequence: TypeAlias = Sequence[int] | Sequence[FloatDataSequence]
```

Comment:
Probably want Sequence[IntDataSequence] on right side?

Reply:
Yes, fixed in fe656689c. Thanks for pointing it out.